loafwallet/file-resources/debug/GoogleService-Info.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@kcw-grunt This is a dummy Google services file?

Reply:
er...its not smart.